docs/topics/collection-ordering.md

```diff
@@ -73,6 +73,45 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
 
+You can also define an order for multiple criteria.
+For example, to sort strings by their length and alphabetically when the lengths are equal, you can write:
+
+```kotlin
+fun main() {
+//sampleStart
+    val sortedStrings = listOf("aaa", "bb", "c", "b", "a", "aa", "ccc")
+        .sortedWith { a, b -> 
+           when (val compareLengths = a.length.compareTo(b.length)) {
+             0 -> a.compareTo(b)
+             else -> compareLengths
+           }
+         }
+
+    println(sortedStrings)
+    // [a, b, c, aa, bb, aaa, ccc]
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+Since sorting by multiple criteria is a very common scenario, the Kotlin standard library provides the [`thenBy()`](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.comparisons/then-by.html) function to simplify it.
+It complements an existing comparator by adding a secondary sorting rule.
```

Comment:
```suggestion
Since sorting by multiple criteria is a common scenario, the Kotlin standard library provides the [`thenBy()`](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.comparisons/then-by.html) function that you can use to add a secondary sorting rule.
```
What do you think about combining these two sentences to bring the explanation of what you can do with the `thenBy()` function a bit sooner?

Reply:
I like it 👍  - let's go with that approach